EIPS/eip-5069.md

```diff
@@ -1,44 +1,118 @@
+
+ - **Provide Legal Advice**: Trademarks, copyrights, patents, prior art, and other legal matters are the responsibility of authors and implementers, not EIP Editors. We are not lawyers, and while we may occasionally make comments touching on these areas, we cannot guarantee any measure of correctness.
+
+Documenting all of the things we would not do is impossible, and the above are just a few examples. We reserve the right to do less work whenever possible!
+
+## Structure
+
+### EIP Editors
+
+We, the Editors, consist of some number of EIP Editors and one Janitor elected by and from the EIP Editors.
+
+EIP Editors are responsible for governing the EIP process itself, electing a Janitor, and stewarding proposals.
+
+The Janitor's two responsibilities (above their EIP Editor duties) are: to determine when rough consensus has been reached on a matter, and determine when/if it is appropriate to re-open an already settled matter.
+
+## Membership
+
+Anyone may apply to join as an EIP Editor. Specific eligibility requirements are left to individual current EIP Editors, but the general requirements are:
+
+ - A strong belief in the above mission;
+ - Proficiency with English (both written and spoken);
+ - Reading and critiquing EIPs;
+ - Participation in governance.
+
+EIP Editors are expected to meet these requirements throughout their tenure, and not doing so is grounds for removal. Any member may delegate some or all of their responsibilities/powers to tools and/or to other people.
+
+## Making Decisions
+
+### Informally
+
+For decisions that are unlikely to be controversial&mdash;especially for decisions affecting a single proposal&mdash;an EIP Editor may choose whatever option they deem appropriate in accordance with our mission.
+
+### Formally
+
+Electing a Janitor, adding/removing EIP Editors, and any possibly-controversial decisions must all be made using variations of this formal process.
+
+#### Preparation
+
+##### Call for Input
+
+For any matter requiring a decision, a call for input must be published in writing to the usual channels frequented by EIP Editors.
+
+##### Quorum
+
+Initially, to establish a valid quorum, all EIP Editors must express their opinion (or vote where appropriate) on the matter under consideration.
+
+After thirty days from the call for input, if not all EIP Editors have responded, the quorum is reduced to the Editors that have responded. This deadline may be extended in exceptional situations.
+
+#### Deciding
+
+##### Electing a Janitor
+
+Any EIP Editor can call for an election for Janitor. The EIP Editor with the most votes once quorum is met is named Janitor until the next election completes. If there is a tie, we'll randomly choose between the EIP Editors with the most votes. Business continues as usual while the election is running.
```

Comment:
```suggestion
Any EIP Editor can call for an election for Janitor. The EIP Editor with the most votes once quorum is met is named Janitor until the next election completes. If there is a tie, the EIP Editors with the most votes will participate in a round-robin cryptographically-secure first-to-one rock-paper-scissors tournament using the following algorithm:

For each pair of participants $(p_1, p_2)$:

1. $p_i$ secretly chooses $c_i$ to be "Rock", "Paper", or "Scissors" and a random string nonce $n_i$.
2. $p_i$ reveals the value of the SHA256 hash of the ASCII-encoded string $c_in_i$ (e.g. "Rock1234".
3. $p_i$ waits for the other player to reveal their hash. If the other player takes more than 10 minutes, $p_i$ wins.
4. Each player reveals $c_i$ and $n_i$. If the other player takes more than 10 minutes, $p_i$ wins.
5. "Rock" beats "Scissors", "Paper" beats "Rock", and "Scissors" beats "Paper". In the case of a draw, GOTO 1.

The EIP Editor with the greatest number of wins wins. In the case of a draw, this process is repeated until there is a winner.

Business continues as usual while the election is running.
```

The algorithm used to randomly determine the winner needs to be fair. I propose this one.